app/controllers/v1/supplemental_claims_controller.rb

```diff
@@ -14,29 +14,38 @@ def show
     def create
       request_body_obj = request_body_hash.is_a?(String) ? JSON.parse(request_body_hash) : request_body_hash
       form4142 = request_body_obj.delete('form4142')
+      sc_evidence = request_body_obj.delete('additionalDocuments')
       sc_response = decision_review_service
                     .create_supplemental_claim(request_body: request_body_obj, user: @current_user)
       submitted_appeal_uuid = sc_response.body.dig('data', 'id')
-      merged_response = {}
-      merged_response[:data] = { body: sc_response.body, status: sc_response.status }
-      unless form4142.nil?
-        form4142_resp = decision_review_service.process_form4142_submission(
-          request_body: request_body_obj, form4142: form4142, user: @current_user, response: sc_response
-        )
-        # We are not using this right now, but keeping it here incase we decide we want to return this
-        # or do some action based on the central mail response, currently we do not.
-        merged_response[:form4142] = { body: form4142_resp.body, status: form4142_resp.status }
+      unless submitted_appeal_uuid.nil?
+        appeal_submission, _ipf_id = clear_in_progress_form(submitted_appeal_uuid)
+        unless form4142.nil?
+          decision_review_service.process_form4142_submission(
+            request_body: request_body_obj, form4142: form4142, user: @current_user, response: sc_response
+          )
+        end
+        submit_evidence(sc_evidence, appeal_submission.id, submitted_appeal_uuid) unless sc_evidence.nil?
```

Comment:
It is getting this value by doing a `request_body_obj.delete('additionalDocuments')` so if it is not provided it will always be `nil` if it IS provided, there is no garuntee that it is not empty, it is whatever the front end is passing in. 

However if the frontend DOES pass in an empty array, it would still be fine because the method is essentially doing a 
``sc_evidence.each`, so if it is empty, it just wont do anything, and also will not error. 

However, I should perhaps add a check that makes sure it is an array from the frontend, but I think the schema checks already enforce that somewhere along the line before this, but I can confirm!